tools/querytee/proxy_endpoint.go

```diff
@@ -224,6 +224,8 @@ func (p *ProxyEndpoint) executeBackendRequests(req *http.Request, resCh chan *ba
 			)
 		}
 
+		relativeDuration := expectedResponse.elapsedTime - actualResponse.elapsedTime
+		p.metrics.relativeDuration.WithLabelValues(req.Method, p.routeName).Observe(relativeDuration.Seconds())
```

Comment:
Do we need to differentiate between HTTP methods here? Performance should be the same regardless of whether it's a GET or a POST request.

Reply:
Good point.